data/jobs.txt

```diff
@@ -2197,6 +2197,26 @@ mission "Colonists [1]"
 		payment 10000
 		dialog "The colonists depart your ship after paying you <payment>."
 
+mission "Stranded field trip to <planet>"
```

Comment:
Don't use substitutions in mission identifiers, use them in the actual `name` property for the template

Reply:
Ah yep, fixed.